apps/api-documenter/src/documenters/MarkdownDocumenter.ts

```diff
@@ -0,0 +1,694 @@
+
+      if (tsdocComment) {
+        // Write the @remarks block
+        if (tsdocComment.remarksBlock) {
+          output.appendNode(new DocHeading({ configuration: this._tsdocConfiguration, title: 'Remarks' }));
+          this._appendSection(output, tsdocComment.remarksBlock.content);
+        }
+
+        // Write the @example blocks
+        const exampleBlocks: DocBlock[] = tsdocComment.customBlocks.filter(x => x.blockTag.tagNameWithUpperCase
+          === StandardTags.example.tagNameWithUpperCase);
+
+        let exampleNumber: number = 1;
+        for (const exampleBlock of exampleBlocks) {
+          const heading: string = exampleBlocks.length > 1 ? `Example ${exampleNumber}` : 'Example';
+
+          output.appendNode(new DocHeading({ configuration: this._tsdocConfiguration, title: heading }));
+
+          this._appendSection(output, exampleBlock.content);
+
+          ++exampleNumber;
+        }
+      }
+    }
+
+    const filename: string = path.join(this._outputFolder, this._getFilenameForApiItem(apiItem));
+    const stringBuilder: StringBuilder = new StringBuilder();
+
+    this._markdownEmitter.emit(stringBuilder, output, {
+      contextApiItem: apiItem,
+      onGetFilenameForApiItem: (apiItemForFilename: ApiItem) => {
+        return this._getLinkFilenameForApiItem(apiItemForFilename);
+      }
+    });
+
+    FileSystem.writeFile(filename, stringBuilder.toString(), {
+      convertLineEndings: NewlineKind.CrLf
+    });
+  }
+
+  /**
+   * GENERATE PAGE: PACKAGE
+   */
+  private _writePackageTables(output: DocSection, apiPackage: ApiPackage): void {
+    const configuration: TSDocConfiguration = this._tsdocConfiguration;
+
+    const namespacesTable: DocTable = new DocTable({ configuration,
+      headerTitles: [ 'Namespace', 'Description' ]
+    });
+
+    const classesTable: DocTable = new DocTable({ configuration,
+      headerTitles: [ 'Class', 'Description' ]
+    });
+
+    const interfacesTable: DocTable = new DocTable({ configuration,
+      headerTitles: [ 'Interface', 'Description' ]
+    });
+
+    const functionsTable: DocTable = new DocTable({ configuration,
+      headerTitles: [ 'Function', 'Description' ]
+    });
+
+    const enumerationsTable: DocTable = new DocTable({ configuration,
+      headerTitles: [ 'Enumeration', 'Description' ]
+    });
+
+    const apiEntryPoint: ApiEntryPoint = apiPackage.entryPoints[0];
+
+    for (const apiMember of apiEntryPoint.members) {
+
+      const row: DocTableRow = new DocTableRow({ configuration }, [
+        this._createTitleCell(apiMember),
+        this._createDescriptionCell(apiMember)
+      ]);
+
+      switch (apiMember.kind) {
+        case ApiItemKind.Class:
+          classesTable.addRow(row);
+          this._writeApiItemPage(apiMember);
+          break;
+
+        case ApiItemKind.Interface:
+          interfacesTable.addRow(row);
+          this._writeApiItemPage(apiMember);
+          break;
+
+/*
+        case 'function':
```

Comment:
Old code?

Reply:
Actually I'll need to finish these before I can merge the PR.  They are causing a regression for the SPFx docs.  I had procrastinated it because I wanted to get feedback about the overall approach first.

---
In reply to: [236360614](https://github.com/Microsoft/web-build-tools/pull/952#discussion_r236360614) [](ancestors = 236360614)

Reply:
#Resolved